.github/workflows/_docs.yml

```diff
@@ -81,10 +81,17 @@ jobs:
       - name: Setup Linux
         uses: ./.github/actions/setup-linux
 
+      - name: Calculate docker image
```

Comment:
Why did you move this to _docs?

Reply:
I'm adding this step to all occurrences of `pull-docker-image` to make sure that pulling docker image would not fail because it couldn't find the image